utils/litemap/src/map.rs

```diff
@@ -139,9 +138,10 @@ impl<K: Ord, V> LiteMap<K, V> {
     /// // not appended since it wasn't in order
     /// assert_eq!(map.get(&2), None);
```

Comment:
Suggestion: Add an example showing that inserting an existing key results in an error.